migrate_pim/migrate_to_saas.rst

```diff
@@ -44,11 +44,83 @@ compare your PIM version (*ex:* ``7.0.123``) with the latest version of Akeneo P
 
 .. tip::
 
-    To check if you need to upgrade, you can scroll to the bottom of your PIM's UI — 
+    To check if you need to upgrade, you can scroll to the bottom of your PIM's UI —
     if the PIM can be upgraded, you will see a message like ``Version: EE 7.0.20 Buckwheat | New patch available: v7.0.29``.
 
 Guides to update the PIM to latest patch can be found here: :doc:`./apply_patch/index`
 
+Check your file structure
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The file structure should be identical to the original PIM file structure.
+
+.. note::
+
+    To get the original PIM file structure, you can install on an empty directory the PIM from the Portal Enterprise Repository.
+    Once you have the ``composer.json`` file in the desired directory, run the following commands:
+
+.. code::
+
+    $ tar xvzf pim-enterprise-standard-<archive-suffix>.tar.gz
+    $ cd pim-enterprise-standard/
+    $ composer --prefer-dist update
+
+
+* There should not be any symbolic link inside the PIM's file structure.
+  If there are symbolic links in your PIM's file structure, we require you to remove them
+  and restore all PIM directories to their original locations.
+
+    The symbolic links might be related to:
+
+        * Release directories
+
+        * Shared asset directories
+
+        * etc.
+
+* The PIM files should be inside the 'pim' directory at the root of ``/home/akeneo``
+
+* Any custom, temporary, or backup directories must be removed from ``/home/akeneo``, as they will not be migrated to Serenity.
+
+Check your storage
+~~~~~~~~~~~~~~~~~~
+
+Please note that only the files from assets and media locations that have been set up in the filesystem adapter will be migrated.
```

Comment:
Please note that only the files from directories that have been set up in the filesystem adapter will be migrated.

Indeed the customer can choose custom directories for import/export files

Reply:
Adapted this sentence in latest commit, please let me know if okay for you